Dockerfile.nginx_rev

```diff
@@ -0,0 +1,10 @@
+# Use const version of nginx in case future versions won't be able to do sth from the bottom.
+FROM nginx:1.23.2
+
+# If ARG won't be passed it should be a hint.
+ARG WWW_SERVER_IP='Ip has not been passed in build docker image.'
+RUN apt-get update
+RUN apt-get install -y systemctl
+RUN apt-get install -y nano
```

Comment:
te nano i nettools są już nie potrzebne w końcowej wersji dockerfile. Byłbym za tym by utrzymywać kontener jak najmniejszy, z jak najmniejszą liczbą instalowanych zależności

Reply:
no ew. ten nano do konfiguracji z readme

Reply:
Może to niepotrzebne już, ale wychodzę z założenia, że jakbym był adminem i deweloper poprosiłby mnie o taki obraz, to takie podstawowe pakiety powinny być.